pageserver/src/http/models.rs

```diff
@@ -0,0 +1,3 @@
+//! If possible, use `::pageserver_api::models` instead.
```

Comment:
This comment probably needs a couple more sentences for the reader to understand when they wouldn't be able to use pageserver_api::models

Reply:
moved to https://github.com/neondatabase/neon/pull/6145/files#r1427989669